lib/turn.rb

```diff
@@ -0,0 +1,78 @@
+class Turn
+  attr_reader :player1, :player2, :spoils_of_war, :type, :winner
+
+  def initialize(player1, player2)
+    @player1 = player1
+    @player2 = player2
+    @spoils_of_war = []
+    @winner = winner
+  end
+
+  def type
+      if player1.deck.rank_of_card_at(0) != player2.deck.rank_of_card_at(0)
+        :basic
+      elsif player1.deck.rank_of_card_at(0) == player2.deck.rank_of_card_at(0) && player1.deck.rank_of_card_at(2) != player2.deck.rank_of_card_at(2)
+        :war
+      elsif player1.deck.rank_of_card_at(0) == player2.deck.rank_of_card_at(0) && player1.deck.rank_of_card_at(2) == player2.deck.rank_of_card_at(2)
+        :mutually_assured_destruction
+      end
+  end
+
+  def winner
+    if type == :basic
+      if player1.deck.rank_of_card_at(0) > player2.deck.rank_of_card_at(0)
+        @player1
+      else
+        @player2
+      end
+      #return the player with the highest ranking card at index position 0
+    elsif type == :war
+      if player1.deck.rank_of_card_at(2) > player2.deck.rank_of_card_at(2)
+        @player1
+      else
+        @player2
+      end
+    elsif type == :mutually_assured_destruction
+      "No Winner"
+    end
+  end
+
+  def pile_cards
+      if type == :basic
+      #each player will send one card (the top card) to the spoils pile
+      # spoils_of_war = []
+      spoils_of_war << player1.deck.remove_card
+      spoils_of_war << player2.deck.remove_card
+      # spoils_of_war
+      end
```

Comment:
Lines 44 & 45 need to be indented one space over but other than that great job using the `remove_card` method that we have defined on the `Deck` class. Another great example of keeping your code DRY!

It's also recommended to remove all your comments when submitting your work. If you want to keep a copy branch that has all the comments on it for your own benefit, that is definitely an option that you can take.